apps/codeforafrica/src/lib/data/rest/index.js

```diff
@@ -37,6 +37,6 @@ export const api = {
 export async function getPageStaticProps(context) {
   const props = await getPageProps(api, context);
   return {
-    props,
+    props: props || {},
```

Comment:
Why are we forcing it here?

Reply:
@kilemensi This is to handle an edge case where the error page has not been created in the CMS yet.